Makefile

```
@@ -3,16 +3,19 @@ RM = rm -f
 MKDIR = mkdir -p -m 0755
 INSTALL = install -o root -g root
 
-all: vmlinux.h https.o malware.lua
+LUA_XDP = http https
```

```suggestion
EBPF_FILTERS = http https
```
Ou algo do gênero.. acho "LUA_XDP` misleading.. parece que é o binding em si